hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -1145,16 +1151,13 @@ public StoreFileWriter createWriterInTmp(long maxKeyCount, Compression.Algorithm
       // if data blocks are to be cached on write
       // during compaction, we should forcefully
       // cache index and bloom blocks as well
-      if (cacheCompactedBlocksOnWrite && totalCompactedFilesSize <= cacheConf
-        .getCacheCompactedBlocksOnWriteThreshold()) {
-        writerCacheConf.enableCacheOnWrite();
+      if (writerCacheConf.enableCacheOnWriteForCompactions(totalCompactedFilesSize)) {
```

Comment:
Hmm.... Looks like you were trying to clean up some old stuff... The weird naming is not yours.

Reply:
Yep, see my reply above.